easylist_cookie/easylist_cookie_specific_block.txt

```diff
@@ -6,6 +6,7 @@
 ||asus.com/js/*/alert-info.js
 ||cmp.infopro-digital.com^
 ||cmp.osano.com^
+||cmp.tori.fi^$domain=tori.fi
```

Comment:
I dont see why you are making this rule this complex.

```
cmp.tori.fi.    300     IN      CNAME   cdn-1047.privacy-mgmt.com
```

I do block all of `privacy-mgmt.com` I do therefor not need this one, so why not just add it as

`||cmp.tori.fi^` ?

Reply:
Chromium may rare use `privacy-mgmt.com` also completly no support CNAME.